Tools/_lust/replace_nt.py

```
@@ -4,32 +4,96 @@
 REPLACEMENTS = {
     r"\bNanotrasen\b": "Qillu",
+    r"\bNanoTrasen\b": "Qillu",
     r"\bNT\b": "Qillu",
     r"\bНТ\b": "Qillu",
     r"\bНанотрейзен\b": "Qillu",
 }
 
+REPLACEMENTS_CI = [(re.compile(p, re.IGNORECASE), r) for p, r in REPLACEMENTS.items()]
+ATTR_RE = re.compile(r'^\s*\.[-\w]+\s*=')
+
+def replace_outside_placeholders(text: str, depth: int = 0):
```

Ruff (0.12.2)

Missing return type annotation for public function `replace_outside_placeholders`

(ANN201)